packages/backend/src/core/discovery/runDiscovery.ts

```diff
@@ -70,7 +71,11 @@ export async function dryRunDiscovery(
   )
 
   if (diff.length > 0) {
-    const messages = diffToMessages(projectConfig.name, diff)
+    const messages = diffToMessages(
```

Comment:
Some performance idea, run findDependants only if the name starts with 'l2beat-'

Reply:
This is happening already, first line inside the `findDependants` function